cmd/headscale/cli/preauthkeys.go

```diff
@@ -107,13 +107,6 @@ var listPreAuthKeys = &cobra.Command{
 				expiration = ColourTime(key.GetExpiration().AsTime())
 			}
 
-			var reusable string
-			if key.GetEphemeral() {
-				reusable = "N/A"
-			} else {
-				reusable = fmt.Sprintf("%v", key.GetReusable())
-			}
```

Comment:
A bit late evening, but I think this `if` is just the wrong way around, it should have the sprintf if it cant get it, and N/A if the node isnt.

I think it should be written like:

```go
reusable := "N/A"
if key.GetEphemeral() {
  reusable = "true" // or "yes"?
}
```

Reply:
This function just returns a boolean though. https://github.com/dustinblackman/headscale/blob/b75e537356c5b712dbae6778c64a1b6f9dbf5217/gen/go/headscale/v1/preauthkey.pb.go#L100-L105

Only chance it's nil is if the `key` is not defined. We could handle all three cases by checking if key is nil.

Reply:
Should we close this instead until this is solved? https://github.com/juanfont/headscale/issues/1712#issuecomment-1938491046